hs/src/Reach/Eval/Core.hs

```diff
@@ -4295,7 +4295,7 @@ assertRefinedArgs :: ClaimType -> [SLSVal] -> SrcLoc -> SLTypeFun -> App (SLVal,
 assertRefinedArgs ct sargs iat (SLTypeFun {..}) = do
   let argvs = map snd sargs
   arges <-
-    mapM (uncurry $ typeCheck_s ct)
+    mapM (uncurry $ typeCheck_s ct Nothing)
```

Comment:
This would be `stf_at` (I think that exists). It is because of the type of refinement. It might be `iat`